### DISCUSSION TITLE
Star button blocking repositories to view on mobile devices

### DISCUSSION THREAD

**Discussion by Salmandabbakuti** [community/community#13427] [Other Feature Feedback, Questions, & Ideas]
![IMG_20220324_200238](https://user-images.githubusercontent.com/29351207/159939676-df2c7ee9-9f3c-4f2d-8f9b-0d2ad77f2de3.jpg)
It looks like ⭐ button on news feed blocking repositories to view on mobile devices

**Comment by arfon** [comment]
Thanks for spotting this @Salmandabbakuti – we'll see what we can do to fix this up!
